dashboard/apps/prototype/models.py

```diff
@@ -141,13 +152,17 @@ class Project(models.Model):
     project_manager = models.ForeignKey(
         'Person', related_name='projects', null=True)
     client = models.ForeignKey('Client', related_name='projects', null=True)
-    discovery_date = models.DateField(null=True)
-    alpha_date = models.DateField(null=True)
-    beta_date = models.DateField(null=True)
-    live_date = models.DateField(null=True)
-    end_date = models.DateField(null=True)
+    discovery_date = models.DateField(null=True, blank=True)
+    alpha_date = models.DateField(null=True, blank=True)
+    beta_date = models.DateField(null=True, blank=True)
+    live_date = models.DateField(null=True, blank=True)
+    end_date = models.DateField(null=True, blank=True)
+    visible = models.BooleanField(default=True)
     raw_data = JSONField(null=True)
 
+    objects = ProjectManager()
```

Comment:
I did it that way originally. I changed it so it could be used as the default ```.objects``` as the filtered results are required everywhere except sync and in the admin. Also for related lookup ```area.projects.filter(...```